site/content/3.13/graphs/graph-visualizer/_index.md

```diff
@@ -0,0 +1,150 @@
+---
+title: Graph Visualizer
+menuTitle: Graph Visualizer
+weight: 85
+description: >-
+  Graph Visualizer allows you to visualize the graphs present in the database.
```

Comment:
- Missing article
- Too vague, need a good value proposition here
- Shouldn't end with a period as per the Readme